### PR TITLE
Add Samyak2 as a maintainer, `chaos-genius` and `toipe` as projects

### PR DESCRIPTION
**Pull request by Samyak2**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/india/blob/main/CONTRIBUTING.md>.

### Which change are you proposing?

  - [x] Adding/updating open source project/s
  - [x] Adding/updating maintainer/s
  
---

### Adding/updating open source project/s

- [x] The project was built in or receives significant contributions from India
- [x] The project has at least 100 stargazers

**Why should this open source project be featured?**

> Please replace this line with an explanation of why you think these changes should be made.

### Adding/updating maintainer/s

- [x] Maintainer's location on their GitHub profile indicates as India
- [x] Maintainer has approved this PR to be included on the website

**What open source project/s does the individual maintain?**

- https://github.com/Samyak2/toipe
- https://github.com/chaos-genius/chaos_genius
- https://github.com/browser-history/browser-history
- https://github.com/pesos/grofer

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing page, or at least not a single one

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: website/data/open-source/projects.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@ Developer Productivity & Tools:
   - meshery/meshery
   - nikochiko/autosaved
 
+Terminal-based Applications:
+  - Samyak2/toipe
+
```
Comment (mvkaran) [inline website/data/open-source/projects.yml:21]: Can you please include this as a part of the larger "Developer Productivity and Tools" category itself currently?

**Review by mvkaran — CHANGES_REQUESTED** [review]
Thanks for your contribution @Samyak2 💙 

Please fix the requested change.

File: website/data/open-source/projects.yml (hunk 1 above)
Comment (Samyak2) [reply website/data/open-source/projects.yml:21]: Sure. I did not add it there because it isn't a developer tool specifically. I'll move it there

Comment (Samyak2) [reply website/data/open-source/projects.yml]: Done

**Comment by Samyak2** [comment]
Rebased on upstream main

**Review by mvkaran — APPROVED** [review]

**Comment by Samyak2** [comment]
Thank you! ❤️

**Pull request merged**